docs/concepts.md

```diff
@@ -0,0 +1,94 @@
+<!--[metadata]>
++++
+title = "Machine concepts and help"
+description = "Understand concepts for Docker Machine, including drivers, base OS, IP addresses, environment variables"
+keywords = ["docker, machine, amazonec2, azure, digitalocean, google, openstack, rackspace, softlayer, virtualbox, vmwarefusion, vmwarevcloudair, vmwarevsphere, exoscale"]
+[menu.main]
+parent="workw_machine"
+weight=3
++++
+<![end-metadata]-->
+
+
+# Understand Machine concepts and get help
+
+Docker Machine allows you to provision Docker on virtual machines that reside
+either on your local system, cloud providers, or on bare metal servers (physical computers). Docker Machine creates a host on a VM, and you use the Docker Engine client as needed to build images and create containers on the host.
+
+## Drivers for creating machines
+
+To create a virtual machine, you supply Docker Machine with the name of the
+driver you want use. The driver determines where the virtual machine is created. For example, on a local Mac or Windows system, the driver is typically Oracle VirtualBox. For provisioning physical machines, a generic driver is provided. For cloud providers, Docker Machine supports drivers such as AWS, Microsoft Azure, Digital Ocean, and many more. The Docker Machine reference includes a complete [list of supported drivers](drivers/index.md).
+
+## Default base operating systems for local and cloud hosts
+
+Since Docker runs on Linux, each VM that Docker Machine provisions relies on a
+base operating system. For convenience, there are default base operating
+systems. For the Oracle Virtual Box driver, this base operating system is the
+`boot2docker.iso`. For drivers used to connect to cloud providers, the base
```

Comment:
I would expect it to say "this base operation system is boot2docker" with a link to https://github.com/boot2docker/boot2docker